.bazelrc

```diff
@@ -28,8 +28,9 @@ build --enable_platform_specific_config=true
 build:linux --cxxopt=-std=c++14
 build:macos --cxxopt=-std=c++14
 
-# Do not create the convenience links, they are inconvenient when the build runs
-# inside a docker image.
+# Do not create the convenience links, they are inconvenient when the build
```

Comment:
> I even fixed some comments for extra consistency.

Thanks.  I hesitate to suggest `s/links, they/links. They/`.  Your choice, depending upon how tired you are of non-functional changes in myriad files.

Reply:
LGTM.